### DISCUSSION TITLE
how to delete a template created while as anonymous user

### DISCUSSION THREAD

**Discussion by adindagadis** [codesandbox/codesandbox-client#5888] [Q&A] [answered]
Help !
I created a template without login and save it and I want to delete it ! I searched for "Delete Template' from the Sandbox info menu but i can't find it. Can you help me to remove it? Thanks

**Comment by andrasbacsai** [comment, marked as answer]
Hey, could you please share the sandbox id with me?

**Reply by adindagadis** [reply]
This issue has been answered via email, thank you
